acceptance/tests/environment/should_find_existing_production_environment.rb

```diff
@@ -0,0 +1,120 @@
+test_name "should find existing production environment"
+tag 'audit:high'
```

Comment:
Not a blocking comment, but I tend to prefer adding integration tests instead of beaker tests whenever possible. Puppet's `spec_helper.rb` creates unique confdir, codedir, etc for each test, so it's ok for the integration test to create a "production" directory without interfering with other tests.